### ISSUE TITLE
how can i disable "Center alignment for image and caption"?

### ISSUE THREAD

**Issue by Alan315123** [SeptemberHX/joplin-plugin-enhancement#28]
thanks for the useful plugin，and is there any way to stop Center alignment for image and caption？cz i wont use it sometimes.

**Comment by SeptemberHX** [comment]
Yes. You can disable it by setting 'Allow to set image size and caption' to false.

**Comment by tylerlekang** [comment]
> Yes. You can disable it by setting 'Allow to set image size and caption' to false.
> 
> But the syntax `[xxx](xxx){width=50%}` will be also disabled. Currently there is no separate setting option for image center alignment for image and caption.

Why??

I can't use this plugin because I don't want images centered, captioned, or numbered. I only want the ability to control image widths/sizes.

Please split out these very different functions!
